form-link/src/main/java/com/ritense/formlink/service/impl/DefaultProcessLinkService.java

```diff
@@ -50,7 +52,7 @@ public DefaultProcessLinkService(
     }
 
     @Override
-    @Transactional
+    @Transactional(isolation = SERIALIZABLE)
```

Comment:
what is this supposed to do, and do we need to test it?

Reply:
When a ednuser clicks really fast on a usertask twice, An error happens and the user task can never be opened again.

Example [user task](https://v10.test.valtimo.nl/dossiers/leningen/document/93e518c7-faef-4654-9f3c-2287726607cb/summary)

Error
`java.lang.IllegalStateException: Multiple form flow instances linked to task: 8fb4b5b9-81d7-11ed-9d07-0242ac110004`


Writing a test for it seems difficult